DataFormats/PatCandidates/src/classes_def_objects.xml

```diff
@@ -289,15 +289,7 @@
   <class name="pat::MET::PackedMETUncertainty" ClassVersion="11">
     <version ClassVersion="11" checksum="3523936012"/>
     <version ClassVersion="10" checksum="1984780659"/>
-    <field name="dpx_" transient="true" />
```

Comment:
Just for my understanding, removing these fields (up to line 300) doesn't break back compatibility? What does `transient="true"` supposed to do? Thanks